draft-ietf-privacypass-protocol.md

```diff
@@ -217,8 +219,8 @@ Consumers of the Issuer directory resource SHOULD follow the usual HTTP caching
 result in use of stale Issuer configuration information, whereas short
 lifetimes may result in decreased performance. When use of an Issuer
 configuration results in token issuance failures, e.g., because the
-configuration information is too stale, the directory SHOULD be fetched and
-revalidated.
+Issuer directory resource is no longer valid and issuance requests using this
```

Comment:
It sounds like you're saying that the directory resource is fresh but invalid?  If so, it would be clearer to make that explicit.
```suggestion
Issuer has invalidated its directory resource before its expiration time and issuance requests using this
```

Reply:
Yes, exactly, thanks for the suggestion!